apps/trackers/constants.py

```diff
@@ -1,9 +1,14 @@
 """
 common tracker constants
 """
+from osidb.helpers import get_env
 
 TRACKERS_API_VERSION = "v1"
 
+# switch to enable or disable the Jira tracker sync
+# - the one controlling the Bugzilla sync is defined in the BBSync app
+SYNC_TO_JIRA = get_env("TRACKERS_SYNC_TO_JIRA", default="False", is_bool=True)
```

Comment:
Please add `TRACKERS_SYNC_TO_JIRA` to `DEVELOP.md` in the .env section.

Reply:
Thank you for the reminder!